docs/source/checks/nlp/data_integrity/plot_frequent_substrings.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+.. _nlp__frequent_substrings:
+
+Frequent Substrings
+********************
+
+This notebook provides an overview for using and understanding the frequent substrings check:
+
+**Structure:**
+
+* `Why check for frequent substrings? <#why-check-for-text-data-duplicates>`__
```

Comment:
```suggestion
* `Why check for frequent substrings? <#why-check-for-frequent-substrings>`__
```
This link won't work unless fixed